reward/stakingInfoCache_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2019 The klaytn Authors
+// This file is part of the klaytn library.
+//
+// The klaytn library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The klaytn library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
+
+package reward
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+// test cache limit of stakingInfoCache
+func TestStakingInfoCache_Add_Limit(t *testing.T) {
+	stakingInfoCache := newStakingInfoCache()
+
+	for i := 1; i <= 10; i++ {
+		testStakingInfo := newEmptyStakingInfo(uint64(i))
```

Comment:
If `newEmptyStakingInfo()` is only for testing, how about moving function to a test file?

Reply:
it is not for testing :)
it is also used in the main code